pavelib/assets.py

```diff
@@ -64,6 +64,9 @@
     'cms': [path('lms') / 'static' / 'sass' / 'partials', ],
 }
 
+# Collectstatic log directory setting
+COLLECTSTATIC_LOG_DIR_ARG = "collect_log_dir"
```

Comment:
I know we like to avoid magic strings, but rather than using strings and `**kwargs` all over the place, you might be better served by making an explicit optional argument for this.